src/components/Modal/Category.vue

```diff
@@ -0,0 +1,117 @@
+<script setup>
+import { ref, computed, toRefs, watchEffect } from 'vue';
+
+const props = defineProps({
+  open: Boolean,
+  categories: Array
+});
+
+const emit = defineEmits(['add', 'close']);
+
+const { open } = toRefs(props);
+
+const categories = computed(() => {
+  return [
```

Comment:
Whats the point of having computed here?

Reply:
@bonustrack before that, there was a function used to increase 1 letter and I cut it out, but I forgot to cut computed. Now fixed.